oidc-api/src/test/java/uk/gov/di/authentication/oidc/services/AccountInterventionServiceTest.java

```diff
@@ -0,0 +1,145 @@
+package uk.gov.di.authentication.oidc.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import uk.gov.di.authentication.oidc.entity.AccountInterventionStatus;
+import uk.gov.di.authentication.oidc.exceptions.AccountInterventionException;
+import uk.gov.di.orchestration.shared.services.ConfigurationService;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+class AccountInterventionServiceTest {
+
+    private final ConfigurationService config = mock(ConfigurationService.class);
+    private final HttpClient httpClient = mock(HttpClient.class);
+
+    private static String ACCOUNT_INTERVENTION_SERVICE_RESPONSE_SUSPEND_REPROVE =
+            """
+            {
+                "intervention": {
+                    "updatedAt": 1696969322935,
+                    "appliedAt": 1696869005821,
+                    "sentAt": 1696869003456,
+                    "description": "AIS_USER_PASSWORD_RESET_AND_IDENTITY_REVERIFIED",
+                    "reprovedIdentityAt": 1696969322935
+                },
+                "state": {
+                    "blocked": false,
+                    "suspended": true,
+                    "reproveIdentity": true,
+                    "resetPassword": false\s
```

Comment:
Why do we need the `\s`?

Reply:
That's weird. I constructed them in another text editor and then pasted them into the text block. Looking at that I can see there was a trailing space which must have been converted during the paste.